README.md

```diff
@@ -1,12 +1,20 @@
 ![Logo](https://avatars3.githubusercontent.com/u/30268214?v=4&s=200)
-[![Build Status](https://travis-ci.org/AlloyTools/org.alloytools.alloy.svg?branch=master)](https://travis-ci.org/AlloyTools/org.alloytools.alloy)
-# Alloy
-
-Alloy 4 is a self-contained executable, which includes the Kodkod
-model finder and a variety of SAT solvers, as well as the standard
-Alloy library and a collection of tutorial examples. The same jar file
-can be incorporated into other applications to use Alloy as an API,
-and includes the source code. See the release notes for details of new
+[![Build Status](https://travis-ci.com/haslab/Electrum2.svg?branch=master)](https://travis-ci.com/haslab/Electrum2)
+# Electrum and Alloy
+
+Electrum is an extension to the Alloy Analyzer by
+[INESC TEC](https://www.inesctec.pt/en) (the Institute for Systems and
+Computer Engineering, Technology and Science) and
+[ONERA](https://www.onera.fr/en) (the French aerospace research center)
+provides an analyzer for Electrum models, a temporal extension to the
```

Comment:
This sentence seems broken.  I guess a "which" or "that" is missing before "provides".

Reply:
Ah, forgot to update the readme, there shouldn't be any reference to Electrum anyway after the PR